docs/experiments-distilbert_kd.md

```diff
@@ -0,0 +1,35 @@
+# Pyserini: Replicating DistilBERT KD Results
+
+## Dense Retrieval
+
+Dense retrieval with distilbert-dot-margin_mse-T2, brute-force index:
```

Comment:
If it's the literal model name, should be in monospace found `distilbert-dot-margin_mse-T2`, otherwise, call it DistilBERT KD in normal font.